src/services/teams/hooks/index.js

```diff
@@ -222,17 +235,20 @@ const filterToRelated = (keys, path, _ifNotLocal = true, objectToFilter) => {
 			path = path.split('.');
 		}
 		const result = objectToFilter || hook;
-		let link; let
-			linkKey;
-		const target = path.length > 0 ? path.reduce((stack, key) => {
-			if (stack[key] === undefined) {
-				throw new NotImplemented('The path do not exist.');
-			}
-			const newTarget = stack[key];
-			link = stack;
-			linkKey = key;
-			return newTarget;
-		}, result) : result;
+		let link;
+		let linkKey;
+		const target =
+			path.length > 0
+				? path.reduce((stack, key) => {
+						if (stack[key] === undefined) {
```

Comment:
Codacy found an issue: [Expected indentation of 5 tabs but found 6.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5989783)